ouroboros-consensus-cardano/tools/db-analyser/Block/Cardano.hs

```diff
@@ -122,12 +128,19 @@ instance HasProtocolInfo (CardanoBlock StandardCrypto) where
     CardanoConfig {..} <- either (error . show) return =<<
       Aeson.eitherDecodeFileStrict' configFile
 
+    -- The paths to the eras configuration files (eg @byronGenesisPath@) are
```

Comment:
I think this change is already merged into `master`.

See my top-level Review comment for more info.

Reply:
Sure. Let's see which conflicts I get when I rebase on top of master :)